apps/saleor-app-checkout/jest.config.js

```diff
@@ -10,14 +11,25 @@ module.exports = {
   collectCoverage: false,
   coverageDirectory: "coverage",
   coverageProvider: "v8",
+  preset: "ts-jest/presets/default-esm",
+  globals: {
+    "ts-jest": {
+      useESM: true,
+      tsconfig: {
+        ...compilerOptions,
+        jsx: "react-jsx",
+      },
+    },
+  },
   moduleNameMapper: {
+    "^(\\.{1,2}/.*)\\.js$": "$1",
```

Comment:
What are we doing here? 🤔

Reply:
Honestly, I'm not 100% sure 🙈. I think it was supposed to strap `.js` from file names - I've tried while I was debugging the issues with `ts-jest` and must have left that by accident. But since we use the preset from `ts-jest` now, it doesn't make sense to have this anymore.

Reply:
I've removed that